tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -599,12 +601,12 @@ class TestGetDagRunsEndDateFilters(TestDagRunEndpoint):
         [
             (
                 f"api/v1/dags/TEST_DAG_ID/dagRuns?end_date_gte="
-                f"{urllib.parse.quote((timezone.utcnow() + timedelta(days=1)).isoformat())}",
+                f"{urllib.parse.quote('2134-10-25T00:55:42.036397+00:00')}",
```

Comment:
Comment for myself: This is interesting one. Pytest-xdist, expects stable set of parameters - they cannot change between threads because it does not not know how to split them,  I have to experiment to see if that works with `id` though (should).